.gitignore

```diff
@@ -22,3 +23,8 @@ dist/
 .vscode
```

Comment:
Can you please move `.vscode` near to `.idea/` as I've grouped all the IDE meta folder together